smoketests/tests/sumo/test_index.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.index import IndexPage
+
+
+class TestIndex:
```

Comment:
Yeah... so pytest supports unittest TestCase, but it gets ornery with fixtures and (imo) doesn't work correctly. Recently, I've decided to nix the TestCase hierarchy and just have it bare.

Also, I used to subclass object because that's how you get new-style objects with Python 2 and that helps with some things. In this case, you don't need anything with new-style objects and the pytest docs have examples like this:

http://pytest.org/latest/getting-started.html#grouping-multiple-tests-in-a-class

If we ever switch to Python 3 (probably unlikely with kitsune, but who knows), everything is new-style.

Anyhow, I could go either way, so let me know what you want me to do.